tests/capture/test_capture_service.c

```diff
@@ -147,23 +149,20 @@ void capture_config(struct capture_conf *config) {
   strcpy(config->middleware_params, "wlan0");
 }
 
-static void test_run_capture(void **state) {
-  (void)state; /* unused */
-
-  struct capture_middleware_context context;
-
-  strcpy(context.ifname, "wlan0");
-  capture_config(&context.config);
+static void test_run_capture_thread(void **state) {
+  (void)state;
 
-  struct eloop_data *eloop = os_zalloc(sizeof(struct eloop_data));
-  assert_non_null(eloop);
+  char *ifname = "wlan0";
```

Comment:
Use the following if this string will be modified:

```suggestion
  char ifname[] = "wlan0";
```

Or use the following if this string will not be modified:

```suggestion
  const char *ifname = "wlan0";
```